perseo-main/src/test/java/com/telefonica/iot/perseo/GenericListenerTest.java

```diff
@@ -31,14 +31,16 @@
 import org.junit.Before;
 import org.junit.BeforeClass;
 import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 
 /**
  *
  * @author brox
  */
 public class GenericListenerTest {
-
+	private static final Logger logger = LoggerFactory.getLogger(GenericListenerTest.class);
```

Comment:
Same here.

Reply:
Fixed in 47ab83625cbfba55660da30a785723e2fd7ecf55

Reply:
Travis ci fixed in b4a4d739cc56604002ce38463fa15aeccb64b823